features/datum.feature

```diff
@@ -98,20 +99,21 @@ Functionaliteit: leveren van een datum
       | burgerservicenummer | 555550001                                      |
       | fields              | burgerservicenummer,datumEersteInschrijvingGBA |
       Dan heeft de persoon met burgerservicenummer '555550001' de volgende gegevens
-      | naam                                | waarde     |
-      | burgerservicenummer                 | 555550001  |
-      | datumEersteInschrijvingGBA.type     | <type>     |
-      | datumEersteInschrijvingGBA.datum    | <datum>    |
-      | datumEersteInschrijvingGBA.jaar     | <jaar>     |
-      | datumEersteInschrijvingGBA.maand    | <maand>    |
-      | datumEersteInschrijvingGBA.onbekend | <onbekend> |
+      | naam                                       | waarde            |
+      | burgerservicenummer                        | 555550001         |
+      | datumEersteInschrijvingGBA.type            | <type>            |
+      | datumEersteInschrijvingGBA.datum           | <datum>           |
+      | datumEersteInschrijvingGBA.jaar            | <jaar>            |
+      | datumEersteInschrijvingGBA.maand           | <maand>           |
+      | datumEersteInschrijvingGBA.onbekend        | <onbekend>        |
+      | datumEersteInschrijvingGBA.leesbaarFormaat | <leesbaarFormaat> |
 
       Voorbeelden:
-      | type           | GbaDatum | datum      | jaar | maand | onbekend |
-      | Datum          | 20200308 | 2020-03-08 |      |       |          |
-      | DatumOnbekend  | 00000000 |            |      |       | true     |
+      | type           | GbaDatum | datum      | jaar | maand | onbekend | leesbaarFormaat
```

Comment:
ook in deze tabel eindigen de rijen niet met een |

Reply:
Aangepast